app/views/org_admin/sections/_section_group.html.erb

```diff
@@ -0,0 +1,14 @@
+<!-- TODO: Move this away from using ID attr -->
+<div class="panel-group section-group"
```

Comment:
We're now going to have 3 `#sections_accordion` will need to see how JS behaves. We should probably switch to a class selector instead